notes/templates/core/index.html

```diff
@@ -0,0 +1,40 @@
+{% load static %}
```

Comment:
You will also want a `base.html` template. Django registration requires it. It would just have the `head` and any style you want to include on every page.